db/seeds.rb

```diff
@@ -326,6 +326,8 @@ def create_if_none(klass, count = nil)
 ##############################################################################
 
 seeder.create_if_none(Badge) do
+  users_in_random_order = User.order(Arel.sql("RANDOM()"))
```

Comment:
I added this line to fix the seed file. I kept getting an error at:
https://github.com/thepracticaldev/dev.to/blob/e13b3de5865ff9f3557f819a66f7ea7bcb70c3ed/db/seeds.rb#L339
With:
```
undefined method limit for NilClass
```

Reply:
this was fixed in https://github.com/thepracticaldev/dev.to/pull/8192/ I believe

Reply:
Forced push with master to get that update, thanks!